### ISSUE TITLE
full e2e testing

### ISSUE THREAD

**Issue by BenTheElder** [kubernetes-sigs/oci-proxy#24]
See also https://github.com/kubernetes-sigs/oci-proxy/issues/23

In addition to integration testing, we should start fully e2e testing a deployed copy of archeio on cloud run in an environment matching what we intend to run at registry.k8s.io.

We could do so by auto-deploying an image to registry-sandbox.k8s.io following merge of changes in this repo.

Probably this is most readily accomplished by adding it to the image building job as an additional step.

We should then make sure we have k8s e2e jobs that use registry-sandbox, and have them in a testgrid dashboard for this project.

This will give us something to look at before promoting changes to registry.k8s.io

**Comment by ameukam** [comment]
/sig k8s-infra

Ref: https://github.com/kubernetes/k8s.io/issues/3411

Requirements:
  - Ensure  continuous deployment for `registry-sandbox.k8s.io`

**Comment by hh** [comment]
This was mentioned as an AI in the March 30th #sig-k8s-infra meeting.

@dims requested one parallel Conformance CI job that uses registry-sandbox.k8s.io  including testgrid dashboard as we want to monitor that the sandbox redirector is production ready.

Do we have an existing job identified or do you want us to create a new one based on an existing one?

registry.k8s.io PROD is a plain redirect, and will will push hard for it to be in everything in the next release cycle.

**Comment by hh** [comment]
@BenTheElder we need a staging deploy that against the staging deploy that is on a dashboard that alerts some of this group. Enough to start. (AGAINST STAGING)

Once we start merging this matching and redirecting code, staging will have the new changes and pre-rollout environment.

Possible another issue about integration testing in presubmit and spin up an integration later.

**Comment by BenTheElder** [comment]
integration is done #23, we maybe have a dashboard? https://github.com/kubernetes/test-infra/pull/25964

need to get alerting, autodeploy.

**Comment by BobyMCbobs** [comment]
> integration is done #23, we maybe have a dashboard? [kubernetes/test-infra#25964](https://github.com/kubernetes/test-infra/pull/25964)
> 
> need to get alerting, autodeploy.

@BenTheElder, autodeploy is currently built but waiting on #41. The permissions for the build account must have access to roles/run.admin to update the services

**Comment by BenTheElder** [comment]
the dashboard is now working and caught #55, thanks all!
